ownership-chain/node/src/chain_spec/mod.rs

```diff
@@ -0,0 +1,26 @@
+use sc_chain_spec::{ChainSpecExtension, ChainSpecGroup};
+use serde::{Deserialize, Serialize};
+
+pub mod klaos;
+pub mod laos;
+
+/// Specialized `ChainSpec` for the normal parachain runtime.
+pub type ChainSpec =
+	sc_service::GenericChainSpec<laos_ownership_runtime::RuntimeGenesisConfig, Extensions>;
```

Comment:
this is used to load chain_spec from file but it has a generic type from laos_ownership_runtime ... is there any way to remove it and make generic ?